vault-core/src/main/java/org/apache/jackrabbit/vault/fs/config/DefaultWorkspaceFilter.java

```diff
@@ -580,27 +581,49 @@ public void dumpCoverage(javax.jcr.Node rootNode, ProgressTrackerListener listen
     /**
      * {@inheritDoc}
      */
-    public void dumpCoverage(Session session, ProgressTrackerListener listener, boolean skipJcrContent)
-            throws RepositoryException {
+    public void dumpCoverage(Session session, ProgressTrackerListener listener, boolean skipJcrContent) throws RepositoryException {
         ProgressTracker tracker = new ProgressTracker(listener);
-        // get common ancestor
-        Tree<PathFilterSet> tree = new Tree<>();
-        for (PathFilterSet set: nodesFilterSets) {
-            tree.put(set.getRoot(), set);
-        }
-        String rootPath = tree.getRootPath();
-        javax.jcr.Node rootNode;
-        if (session.nodeExists(rootPath)) {
-            rootNode = session.getNode(rootPath);
-        } else if (session.nodeExists("/")) {
-            log.warn("Common ancestor {} not found. Using root node", rootPath);
-            rootNode = session.getRootNode();
-            rootPath = "/";
-        } else {
-            throw new PathNotFoundException("Common ancestor " + rootPath+ " not found.");
-        }
-        log.debug("Starting coverage dump at {} (skipJcrContent={})", rootPath, skipJcrContent);
-        dumpCoverage(rootNode, tracker, skipJcrContent);
+
+        List<javax.jcr.Node> nodes = new ArrayList<>();
```

Comment:
What is this variable being used for?

Reply:
leftover from refactoring; thanks for spotting.